price_feeder/tasks.py

```diff
@@ -326,16 +326,22 @@ def price_from_sources(self):
 
         return result
 
-    def post_price(self, price_no_precision, info_contracts, re_post, task=None, global_manager=None):
-
+    def post_price(self, price_no_precision, info_contracts, post, re_post, task=None, global_manager=None):
```

Comment:
I like more the "re_post_in_range" expressiveness of this variable, as to this point we don't actually know it is a re_post, but just that the price comparison condition allows it. WDYT?